ngrok/kubernetes-ingress-controller
# root-cas setting

closes: #369 

## What
Utilizes the `trusted` or `host` CA as the source of truth via a helm install flag.

## How
Takes an install option for `--set rootCAs=host`  and plumb the `isHostCA` check into the `caCerts` for it to just get the host certs. 

## Verification

- Build 2048 image, copying in cert and updating certs, apply changes
- Set `--set rootCAs host` and make deploy, no controller errors ✅ and can play example
- Set `--set rootCAs host` with canonical 2048 example without cert gives errors in controller
- Set `--set rootCAs trusted` no errors and plays 2048 as expected
- no set plays as expected
